arviz/data/io_dict.py

```diff
@@ -115,6 +137,26 @@ def observed_data_to_xarray(self):
             observed_data[key] = xr.DataArray(vals, dims=val_dims, coords=coords)
         return xr.Dataset(data_vars=observed_data, attrs=make_attrs(library=None))
 
+    @requires("constant_data")
+    def constant_data_to_xarray(self):
+        """Convert constant_data to xarray."""
+        data = self.constant_data
+        if not isinstance(data, dict):
+            raise TypeError("DictConverter.constant_data is not a dictionary")
+        if self.dims is None:
+            dims = {}
+        else:
+            dims = self.dims
+        constant_data = dict()
+        for key, vals in data.items():
+            vals = np.atleast_1d(vals)
```

Comment:
Why `atleast_1d`? Woudn't a scalar also be reasonable?

Reply:
I copied the code from `observed_data_to_xarray` above without thinking too much about it. I will check how are scalar values stored in `observed_data` and `constant_data` groups and if there are extra dimension I will modify both.